samples/central-login/README.md

```diff
@@ -0,0 +1,39 @@
+# Central login
+
+The ForgeRock SDK provides an option for using the Authorization Code Flow (with PKCE) with a centralized login application. For a non-authenticated user, use the login of "redirect" option (from the `TokenManager`) to request OAuth/OIDC tokens. This instructs the SDK to redirect the user to the login application that uses the ForgeRock platform. After successful authentication, the SDK redirects the user back to the original application to obtain OAuth/OIDC tokens and complete the centralized login flow.
```

Comment:
This is better, but I'd like to make a few more tweaks:

The ForgeRock SDK provides an option to use a redirect flow for authenticating users, rather than embedding the login form into your app. This uses the traditional Authorization Code Flow (with PKCE) with a centralized login application. To do this, add the property of `login` with the value of `"redirect"` to the `TokenManager.getTokens` options object. This instructs the SDK to redirect the user to a dedicated login application. After successful authentication, the SDK redirects the user back to the original application to obtain OAuth/OIDC tokens and complete the centralized login flow.